app/components/plaintext_message/plaintext_message.html.erb

```diff
@@ -1 +1 @@
-<div <%= attrs %>><%= rendered if !empty? %></div>
+<div <%= attrs %>><%= rendered.html_safe if !empty? %></div>
```

Comment:
Let's favor semantic HTML?

`rendered` is not HTML, from what I can tell, so this is not a wrapper div for some combination of allowed HTML tags, correct? If it is plain text, as is stored in `request.text`, then I guess the most semantic tag to use would be a `<p>`.

From [MDN docs on %div usage](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div#usage_notes):

> The %div element should be used only when no other semantic element (such as [%article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) or [%nav](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)) is appropriate.

Whereas [MDN docs on %p](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p):

> The %p [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a paragraph.

Since, we don't use an editor, there is always the possibility that the editor adds several paragraphs, instead of one, but it is still more semantic than a `<div>` tag.

Reply:
> rendered is not HTML, from what I can tell, so this is not a wrapper div for some combination of allowed HTML tags, correct?

It is! It will contain a `<p>` tag for every paragraph in the request text. For example, if you had a request like this:

```
Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.

Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem.
```

It would be rendered like this:

```
<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.</p>

<p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem.</p>
```

Relevant part in the code base: https://github.com/tactilenews/100eyes/blob/main/app/components/plaintext_message/plaintext_message.rb#L20

Reply:
Right, we use [simple_format](https://apidock.com/rails/v6.1.3.1/ActionView/Helpers/TextHelper/simple_format), I saw that, but forgot what it did.